src/Data/Sum/Properties.agda

```diff
@@ -113,3 +115,6 @@ map₂-cong : {g g′ : C → D} →
             g ≗ g′ →
             map₂ {A = A} g ≗ map₂ g′
 map₂-cong g≗g′ = [,-]-cong ((cong inj₂) ∘ g≗g′)
+
+swap↔ : (A ⊎ B) ↔ (B ⊎ A)
```

Comment:
I think this should be named `swap-↔`. Also it would be great if you could move it next to the `swap-involutive` lemma!